sdk/communication/azure-communication-sms/src/main/java/com/azure/communication/sms/SmsClient.java

```diff
@@ -66,34 +62,48 @@ public SendSmsResponse sendMessage(PhoneNumberIdentifier from, List<PhoneNumberI
      * @param from Number that is sending the message.
      * @param to The recipient's phone number.
      * @param message message to send to recipient.
+     * @param smsOptions set options on the SMS request, like enable delivery report, which sends a report
+     *                   for this message to the Azure Resource Event Grid.
+     * @param context context to use
      * @return response for a successful send Sms request.
      */
-    @ServiceMethod(returns = ReturnType.SINGLE)    
-    public SendSmsResponse sendMessage(PhoneNumberIdentifier from, PhoneNumberIdentifier to, String message) {
-        if (to == null) {
-            logger.logThrowableAsError(new NullPointerException("Argument 'to' cannot be null."));
-        }
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public SmsSendResult send(String from, String to, String message,
+                              SmsSendOptions smsOptions, Context context) {
```

Comment:
Does this need to be on second line?

Reply:
no, I thought it was going to cause a checkstyle issue but it did not J.